src/components/Participant/RegistrationProcess/StartRegistrationProcess/StartRegistrationProcess.tsx

```diff
@@ -2,13 +2,40 @@ import DocumentTitle from "react-document-title";
 import StepWrapper from "../../../Shared/StepWrapper/StepWrapper";
 import { ContentContext } from "../../../../context/ContentContext";
 import { useContext } from "react";
+import { Stack } from "@mui/material";
+import DTEButton from "../../../Shared/UI/DTEButton/DTEButton";
+import { useHistory } from "react-router-dom";
+import NhsLoginButton from "../../../Shared/UI/NhsLoginButton";
 
 function StartRegistrationProcess() {
   const { content } = useContext(ContentContext);
+  const history = useHistory();
+
+  const handleGoBack = () => {
+    history.goBack();
+  };
 
   return (
     <DocumentTitle title={content["register-page-document-title"]}>
-      <StepWrapper>{content["register-page"]}</StepWrapper>
+      <StepWrapper>
+        {content["register-page"]}
+        <Stack spacing={2} direction="row">
+          <NhsLoginButton buttonText="Continue to NHS login" />
+          <a href="https://volunteer.bepartofresearch.nihr.ac.uk/Participants/Register/Questions">
```

Comment:
Anchor tags <a> will work but they are not the best performance for react.  Always use the router that comes with React.  In this case use the onClick method on the DTEButton and use history

`onClick={() => history.push("/Participants/Register/Questions")}`